cg/meta/workflow/jasen.py

```diff
@@ -0,0 +1,20 @@
+"""Module for Jasen Analysis API."""
+
+import logging
+
+from cg.constants import Workflow
+from cg.meta.workflow.nf_analysis import NfAnalysisAPI
+from cg.models.cg_config import CGConfig
+
+LOG = logging.getLogger(__name__)
+
+
+class JasenAnalysisAPI(NfAnalysisAPI):
+    """Handles communication between Jasen processes and the rest of CG infrastructure."""
+
+    def __init__(
+        self,
+        config: CGConfig,
+        workflow: Workflow = Workflow.JASEN,
```

Comment:
does this need to be a parameter since it will always be `Workflow.JASEN`?

Reply:
I see that all the other workflows do this, and was assuming it is to conform to a common interface?